wrapper/src/types/serverMonitors.ts

```diff
@@ -0,0 +1,12 @@
+export interface ServerMonitor {
+  id: string;
+  name: string;
+  creatorId: string;
+  collectionId: string;
+}
+
+export interface NewServerMonitor {
+  name: string;
+  creatorId: string;
```

Comment:
This shouldn't be included because the creatorId is pulled from the auth token

Reply:
ok, I'll take it out. also, what about collection id? is it mandatory?

Reply:
It is mandatory because the collection ID is used to link the service to it collection, like categorization.